### ISSUE TITLE
Input form not applying styles 0.6.1

### ISSUE THREAD

**Issue by leonardseymore** [angular/material#1191]
Good day

I have tried copying the input demo code.  All other controls seem to render correctly, only the input form doesn't have any styles applied.

![screenshot from 2015-01-14 15 24 01](https://cloud.githubusercontent.com/assets/1201248/5739739/69c52d3a-9c05-11e4-96ec-65c2535c71bb.png)

```xml
<md-content md-scroll-y flex class="md-padding">
  <div layout="column">
    <form name="userForm">
    <md-content class="md-padding" layout="row" layout-sm="column">
      <md-input-container>
        <label>Title</label>
        <input ng-model="user.title">
      </md-input-container>
      <md-input-container>
        <label>Email</label>
        <input ng-model="user.email" type="email">
      </md-input-container>
    </md-content>

    <md-content class="md-padding">


        <md-input-container flex>
          <label>Company (Disabled)</label>
          <input ng-model="user.company" disabled>
        </md-input-container>

        <div layout layout-sm="column">
          <md-input-container flex>
            <label>First Name</label>
            <input ng-model="user.firstName">
          </md-input-container>
          <md-input-container flex>
            <label>Last Name</label>
            <input ng-model="theMax">
          </md-input-container>
        </div>

        <md-input-container flex>
          <label>Address</label>
          <input ng-model="user.address">
        </md-input-container>

        <div layout layout-sm="column">
          <md-input-container flex>
            <label>City</label>
            <input ng-model="user.city">
          </md-input-container>
          <md-input-container flex>
            <label>State</label>
            <input ng-model="user.state">
          </md-input-container>
          <md-input-container flex>
            <label>Postal Code</label>
            <input ng-model="user.postalCode">
          </md-input-container>
        </div>

        <md-input-container flex>
          <label>Biography</label>
          <textarea ng-model="user.biography" columns="1" md-maxlength="150"></textarea>
        </md-input-container>

    </md-content>

    </form>
  </div>
</md-content>
```

Thanks a lot

**Comment by PaulMougel** [comment]
As you can see on the landing page of the documentation:
![image](https://cloud.githubusercontent.com/assets/1788596/5739784/8eb6c4e4-9bfd-11e4-94c2-1c2e3deec3aa.png)
the docs are built for a specific commit, in this case [v0.7.0-rc2 - SHA 1981ac9 ](https://github.com/angular/material/commit/1981ac9790359e1ba85d7f61a7b4d8af8a8dc431). You're reading a newer version of the docs, which explains why the form elements don't work.

If you want to read the documentation for a particular version, you can [build it yourself](https://github.com/angular/material/blob/master/docs/README.md).

In your particular case, `md-text-float` was deprecated by this commit: 60fcd6f4d8b895162b37a940c3f33164d8044382, which is only available in 0.7.0-rc2 and upwards.

You can either upgrade your angular-material version, read the commit message to see how `md-text-float` was used before the change, or build the documentation for your particular version.

**Comment by leonardseymore** [comment]
Excellent, thanks for the explanation and fast response!
